test/fixtures/basic/package.json

```diff
@@ -4,6 +4,7 @@
     "run-series": "^1.1.1"
   },
   "devDependencies": {
+    "coffee-script": "^1.10.0",
```

Comment:
`tape` was actually my first attempt. It has many dependencies which caused several tests to time out.